tally/TrackLengthMeshTally.cpp

```diff
@@ -272,13 +276,31 @@ void TrackLengthMeshTally::write_data(double num_histories) {
         rel_err = sqrt((error / (tally * tally)) - (1. / num_histories));
       }
 
-      rval = mb->tag_set_data(error_tags[j], &t, 1, &rel_err);
-      assert(rval == MB_SUCCESS);
+      tally_vect[j] = score;
+      error_vect[j] = rel_err;
+
+      // TODO: It would be convenient for users to be able to get the total
```

Comment:
But this version doesn't include the total at all... right?  Couldn't we just add an additional tag that is also the total?  In fact, you create the `total_tally_tag` above, but don't use it.